app/controllers/concerns/create_session.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module CreateSession
+  extend ActiveSupport::Concern
+  require 'json_web_token'
```

Comment:
You would need to add the gem or use the methods provided by the [jwt gem](https://github.com/jwt/ruby-jwt).